amqp/src/main/resources/amqp.conf

```diff
@@ -20,7 +20,7 @@ config.amqp = {
   consumerThreadPoolQueueCapacity = -1
 
   # Interval of the heartbeats that are used to detect broken connections. Zero for none. See for more details: https://www.rabbitmq.com/heartbeats.html
-  heartbeatIntervalSec = 1
+  heartbeatIntervalSec = 30
```

Comment:
The application can't deliver heartbeats each second under a heavy load, so RabbitMQ closes a connection on a missing heartbeat. AFAIR, we faced this issue before during our MS load testing. So I'm considering the previous 1s default value as dangerous and harmful. RabbitMQ defaults are also far from 1s: "The timeout is in seconds, and default value is 60 (580 prior to release 3.5.5).".